src/pages/patientView/PatientViewPage.tsx

```diff
@@ -112,6 +124,13 @@ export default class PatientViewPage extends React.Component<
         );
         getBrowserWindow().patientViewPageStore = this.patientViewPageStore;
 
+        const openResourceId =
```

Comment:
maybe add this as constructor parameter of store?

Reply:
but then it would be the only parameter that we pass directly into the store rather than setting later